pallets/aleph/src/lib.rs

```diff
@@ -132,21 +132,15 @@ pub mod pallet {
         fn get_authorities_for_next_session(
             next_authorities: Vec<(&T::AccountId, T::AuthorityId)>,
         ) -> Vec<T::AuthorityId> {
-            let next_committee_ids: BTreeSet<_> =
-                NextFinalityCommittee::<T>::get().into_iter().collect();
-
-            let next_committee_authorities: Vec<_> = next_authorities
+            let mut account_to_authority: BTreeMap<_, _> = next_authorities.into_iter().collect();
+            let stored_accounts = NextFinalityCommittee::<T>::get();
```

Comment:
nit: I would leave the name next_committee_ids here.

Reply:
how about `next_committee_accounts`? To emphasize distinction between `AccountId` and `AuthorityId`

Reply:
:+1: